hack/data-plane.sh

```diff
@@ -132,9 +135,16 @@ function data_plane_unit_tests() {
   return $mvn_output
 }
 
+function tls_setup() {
+	ko resolve ${KO_FLAGS} -Rf ${BROKER_TLS_CONFIG_DIR} \
+		-Rf ${SINK_TLS_CONFIG_DIR} \
+		-Rf ${CHANNEL_TLS_CONFIG_DIR} \
+		| "${LABEL_YAML_CMD[@]}" >"${EVENTING_KAFKA_TLS_NETWORK_ARTIFACT}"
```

Comment:
Did you consider just adding the `ko resolve ...` command to the `k8s` function since there we create the various artifacts? 

Bash functions are public and global, so they might easily clash with one another, so we either give it a specific and hard to conflict name or we just to don't define a function

Reply:
Oh I didn't know they were global! I felt like the tls stuff was logically it's own idea, but I see why it would make sense to put it in the `k8s` function. I'll push a commit to do that.